pallets/nft/src/tests.rs

```diff
@@ -144,43 +152,9 @@ fn invalid_proof() {
 				),
 				Error::<MockRuntime>::InvalidProofs
 			);
-		})
-}
-
-#[test]
-fn insufficient_balance_to_mint() {
```

Comment:
I remove the whole test because here we are **only** testing if the `fee_to_burn()` call inside the `NFT::validate_mind()` is working well. This is something already tested in pallet_fees. Also, because the pallet_fee is mocked at this test, no longer returns an `InsufficienteBalance` error.